src/org/opendatakit/briefcase/push/aggregate/PushToAggregate.java

```diff
@@ -164,7 +165,8 @@ void pushFormAndAttachments(FormStatus form, List<Path> attachments, RunnerStatu
     }
 
     tracker.trackStartSendingFormAndAttachments(part, parts);
-    Response response = http.execute(server.getPushFormRequest(form.getFormFile(briefcaseDir), attachments));
+    Response<InputStream> response = http.execute(server.getPushFormRequest(form.getFormFile(briefcaseDir), attachments));
+
```

Comment:
I think these are leftover changes from a previous stage of the PR. We don't need the type parameter here anymore.
```suggestion
    Response response = http.execute(server.getPushFormRequest(form.getFormFile(briefcaseDir), attachments));
```